SpatialGDK/Extras/schema/server_worker.schema

```diff
@@ -7,7 +7,7 @@ import "unreal/gdk/spawner.schema";
 type ForwardSpawnPlayerRequest {
     SpawnPlayerRequest spawn_player_request = 1;
     UnrealObjectRef player_start = 2;
-    string client_worker_id = 3;
+    EntityId client_worker_id = 3;
```

Comment:
If you're not sending the worker system entity ID instead of the worker id string, we should rename this prop accordingly

Reply:
Below it's `EntityId client_system_entity_id = 5;`

Reply:
This is the client worker entity id - this name seems pretty sufficient to me. What do you want to call it?

Reply:
I'm not sure how I didn't see your second comment